core/src/smartcontracts/isi/account.rs

```diff
@@ -261,13 +261,6 @@ pub mod isi {
             // Check if account exists
             wsv.account_mut(&account_id)?;
 
-            let definition = wsv
-                .permission_token_definitions()
-                .get(&permission_id)
-                .ok_or_else(|| FindError::PermissionTokenDefinition(permission_id.clone()))?;
-
-            permissions::check_permission_token_parameters(&permission, definition)?;
```

Comment:
Also shouldn't we check strings now?

Reply:
the argument in the same as [here](https://github.com/Arjentix/iroha/pull/1/files#r1234300430).